tests/extra/test_job_management.py

```diff
@@ -0,0 +1,161 @@
+import os
+from pathlib import Path
+
+import pandas as pd
+import pytest
+import shapely.geometry.point as shpt
+
+import openeo
+from openeo.extra.job_management import MultiBackendJobManager
+from openeo.rest.job import RESTJob
+
+
+@pytest.fixture
+def temp_working_dir(tmp_path):
+    """Use a temporary working directory, because some tests generate files
+    in the working directory, and that pollutes your git work tree.
+
+    Preventing it would be better, but sometimes that is difficult.
+    TODO: What if we want to keep those files for inspecting after the test run, but preferably in a separate folder?
+    """
+
+    orig_workdir: Path = Path.cwd()
+    # Make the CWD a bit more specific because test can use tmp_dir for other things too.
+    temp_word_dir: Path = tmp_path / "test-workdir"
+    if not temp_word_dir.exists():
+        temp_word_dir.mkdir()
+    os.chdir(temp_word_dir)
```

Comment:
Having to use `os.chdir` juggling from all tests feels like code smell to me. 
As mentioned above, I would make sure `MultiBackendJobManager` takes a root/working dir option to work in

Reply:
Indeed this workaround is a code smell. We need to be able to set the directory where it stores files.

Reply:
Now the job manager support a configurable root directory.
Removed the fixture that wrangles the current working.